libvmi/driver/kvm/kvm_events.c

```diff
@@ -135,7 +135,7 @@ process_cb_response_emulate(
                             rpl->pf.ctx_size = libvmi_event->emul_read->size;
                             // set linear address
                             // TODO: ARM support
-                            rpl->pf.ctx_addr = libvmi_event->x86_regs->rip;
+                            rpl->pf.ctx_addr = libvmi_event->mem_event.gla;
```

Comment:
This seems to be an unrelated fix for a bug.

Reply:
right, i will make a separate PR